src/help/help_topic_generators.cpp

```diff
@@ -650,35 +671,31 @@ std::string unit_topic_generator::operator()() const {
 			}
 
 			if (info.union_type().size() == 1 && info.union_type()[0] == info.number() && info.is_nonnull()) {
-				terrain_moves.push_back(std::make_tuple(info.name(), // 0 name (t_string)
+				terrain_movement_info movement_info = 
+				{
+					info.name(), // 0 name (t_string)
```

Comment:
You don't need the indices in the comments now when the fields are accessed by their names.